packages/ts-transform/src/utils/visit-source-file-ensure-style-import.tsx

```diff
@@ -3,61 +3,92 @@ import * as log from './log';
 import * as constants from '../constants';
 
 const COMPILED_PKG = '@compiled/css-in-js';
+const COMPILED_STYLE_PKG = '@compiled/style';
 
 interface Opts {
   imports?: string[];
   removeNamedImport?: string;
 }
 
+const isStylePkgFound = (sourceFile: ts.SourceFile | ts.ModuleBlock): boolean => {
+  return !!sourceFile.statements.find(
+    (statement: ts.Node) =>
+      ts.isImportDeclaration(statement) &&
+      ts.isStringLiteral(statement.moduleSpecifier) &&
+      statement.moduleSpecifier.text.startsWith(COMPILED_STYLE_PKG)
+  );
+};
+
 export const visitSourceFileEnsureStyleImport = (
   sourceFile: ts.SourceFile,
   context: ts.TransformationContext,
   {
-    imports = [constants.COMPILED_STYLE_COMPONENT_NAME, constants.COMPILED_COMPONENT_NAME],
+    imports = [constants.COMPILED_COMPONENT_NAME, constants.COMPILED_STYLE_COMPONENT_NAME],
     removeNamedImport,
   }: Opts = {}
 ): ts.SourceFile => {
-  const visitor = (node: ts.Node): ts.Node => {
+  const visitor = (node: ts.Node): ts.Node | Array<ts.Node> | undefined => {
     if (
       ts.isImportDeclaration(node) &&
       ts.isStringLiteral(node.moduleSpecifier) &&
       node.moduleSpecifier.text === COMPILED_PKG
     ) {
       log.log('ensuring style export is defined');
+      // if a style import doesn't exist, add it as a sibling to this node
+      // idealy this should occur in a separate transform, possibly as an independent transform
+      // before any of the three processes start.
+
+      // if it exists already, then just remove the removedImports from the importstatement.
 
       const defaultImport = node.importClause && node.importClause.name;
```

Comment:
nit: I think we can get rid of this now there isn't a default import on css-in-js